doc/classes/Directory.xml

```diff
@@ -108,6 +108,13 @@
 				Returns the currently opened directory's drive index. See [method get_drive] to convert returned index to the name of the drive.
 			</description>
 		</method>
+		<method name="get_directories">
+			<return type="PackedStringArray" />
+			<description>
+				Returns a [PackedStringArray] containing filenames of the directory contents, excluding files.
+				Affected by [member include_hidden] and [member include_navigational.
```

Comment:
```suggestion
				Affected by [member include_hidden] and [member include_navigational].
```

Reply:
`doc/tools/make_rst.py` didn't report it.